tabpy-server/setup.py

```diff
@@ -36,7 +34,7 @@
         'decorator',
         'future',
         'genson',
-        'jsonschema>=2.3.0',
+        'jsonschema~=2.3.0',
```

Comment:
What does unary inversion do here?

Reply:
`~=` means `compatible with. We had issues with `>=` when one of the environments had a newer version of the package and Tornado started throwing exceptions complaining about missing attributes.